WebHostLib/upload.py

```diff
@@ -20,6 +20,43 @@
 
 banned_zip_contents = (".sfc", ".z64", ".n64", ".sms", ".gb")
 
+def parse_multidata(compressed_multidata, files={}):
+    decompressed_multidata = MultiServer.Context.decompress(compressed_multidata)
+
+    slots: typing.Set[Slot] = set()
+    recompress = False
```

Comment:
If you remove the conditional that depends on recompress, please also remove the variable.
```suggestion
```